### PR TITLE
Issue #1776 Updating Origin version to v3.7.0

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
retest this please

**Comment by budhrg** [comment]
retest this please

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # Various versions - Minishift, default OpenShift, default B2D ISO
 MINISHIFT_VERSION = 1.9.0
-OPENSHIFT_VERSION = v3.6.0
+OPENSHIFT_VERSION = v3.7.0
```
Comment (praveenkumar) [inline Makefile]: 👍

**Comment by hferentschik** [comment]
retest this please

**Comment by LalatenduMohanty** [comment]
> If we only do this change then we are breaking our master to use < 3.7.0 version of OpenShift. Because as per Ben currently, you can't provision a 3.6 cluster using 3.7 client binary and in our code we only download oc client binary if it is greater than our specified version, otherwise we use same client to provision.

Good point, I will fix it with #1417

**Comment by hferentschik** [comment]
There is also the service catalog question. Per 3.7 it should be enabled er default. Does this work? With this upgrade did the service catalog get provisioned automatically? If so, there is no need anymore for the experimental flag and it should be removed from the code and docs.

**Pull request closed without merging**